addOns/pscanrulesAlpha/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## [35] - 2021-11-07
+### Fixed
+- Add info alerts for each scripts found along with the integrity hash in subresource intergrity attribute scan rule (Issue 5894).
```

Comment:
Should be in "unreleased".

The version and date as set by the release process.

Reply:
Is it correct now?

Reply:
Could have left the fixed heading 😉

Reply:
Okay :)